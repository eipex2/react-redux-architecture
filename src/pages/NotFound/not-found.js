import React from 'react'

import { Page } from '../../components'

import "./not-found.css"

/**
 * a `NotFound` page to handle all unvailable routes
 * @param props 
 */
const NotFound = (props) => (
    <Page className="not_found">
        <h1>404</h1>
        <h3>PAGE NOT FOUND</h3>
    </Page>
)


export default NotFound