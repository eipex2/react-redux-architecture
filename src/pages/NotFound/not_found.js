import React from 'react'

import { Page } from '../../components'

import "./not_found.css"
const NotFound = () => {
    return (
        <Page className="not_found">
            <h1>404</h1>
            <h3>PAGE NOT FOUND</h3>
        </Page>
    )
}

export default NotFound