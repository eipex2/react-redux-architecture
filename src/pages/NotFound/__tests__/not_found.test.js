import React from 'react'
import { shallow } from 'enzyme'
import NotFound from '../not_found'

describe('[<NotFound />', () => {

    it('shd render NotFound page', () => {
        const wrapper = shallow(<NotFound />)

        expect(wrapper.getElement()).toMatchSnapshot()
    })
})