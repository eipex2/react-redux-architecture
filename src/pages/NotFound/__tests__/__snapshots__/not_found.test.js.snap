// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<NotFound /> shd render NotFound page 1`] = `
<Page
  className="not_found"
>
  <h1>
    404
  </h1>
  <h3>
    PAGE NOT FOUND
  </h3>
</Page>
`;
